src/components/Queens/Queen.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Queen = ({ name, image, url }) => (
+  <section>
+    <a href={url}>{name}</a>
```

Comment:
You're getting a warning about having and `<a>` tag within an `<a>` tag. The reason for that is because the `<Link />` component from React Router actually uses `<a>` tags to redirect you so you don't actually need an `<a>` tag here.